index.js

```diff
@@ -68,6 +69,20 @@ module.exports = function(input, inputMap) {
 	function untouched() {
 		callback(null, input, inputMap);
 	}
+	function fetchHttp(source, callback) {
+		http.get(source, function (resp) {
```

Comment:
Built-in `http.get()` is very bare-bones, e.g. doesn't follow redirects. Should probably use some higher-level library.
Also, does this need caching? Or is there already a cache on another layer?

Reply:
you can use `got`

Reply:
> Built-in `http.get()` is very bare-bones, e.g. doesn't follow redirects. Should probably use some higher-level library.
> Also, does this need caching? Or is there already a cache on another layer?

I understand. Any considerations about bringing in a new dependency?

Reply:
> you can use `got`

I'll try. Thanks for recommendation.